sql/core/src/test/resources/sql-tests/results/table-valued-functions.sql.out

```diff
@@ -83,8 +83,13 @@ select * from range(1, null)
 -- !query 6 schema
 struct<>
 -- !query 6 output
-java.lang.IllegalArgumentException
-Invalid arguments for resolved function: 1, null
+org.apache.spark.sql.AnalysisException
+error: table-valued function range with alternatives:
```

Comment:
@rxin will it be better to display the function name within single quotes ?

Reply:
probably could go either way

Reply:
@rxin Ok.